zigzag/module_hierarchy_facade.py

```diff
@@ -32,7 +33,11 @@ def get_module_hierarchy(self, classname):
             list[str]: The strings to use for the module_hierarchy
         """
 
-        if self._mediator.ci_environment == 'asc':
+        if 'module_hierarchy' in self._mediator._config_dict.keys():
+            defined_hierarchy = ast.literal_eval(self._mediator._config_dict['module_hierarchy'])
```

Comment:
Eek! Why are we using a literal eval? We should just validate that the config file complies with a schema.